tests/test_series.py

```diff
@@ -556,7 +556,10 @@ def test_types_idxmax() -> None:
 
 def test_types_value_counts() -> None:
     s = pd.Series(["a", "b"])
-    check(assert_type(s.value_counts(), "pd.Series[int]"), pd.Series, np.integer)
+    c1: pd.Series[int] = s.value_counts()
+    check(c1, pd.Series, np.integer)
+    c2: pd.Series[float] = s.value_counts(normalize=True)
+    check(c2, pd.Series, float)
```

Comment:
And here too with `check(assert_type(...`

Reply:
Done all three